sympy/concrete/tests/test_sums_products.py

```diff
@@ -402,9 +402,9 @@ def check_exact(f, a, b, m, n):
     # Not exact
     assert Sum(k**6, (k, a, b)).euler_maclaurin(0, 2)[1] != 0
     # Numerical test
-    for m, n in [(2, 4), (2, 20), (10, 20), (18, 20)]:
+    for mi, n in [(2, 4), (2, 20), (10, 20), (18, 20)]:
```

Comment:
This looks like a strange change in that only one of the two variables is changed. I'm not too worried because it is in a test file.

Reply:
I think it's a bug in flake8 that it only reports a problem for `m`. Both of these are shadowing the global `m` and `n` at the top of the file.

Reply:
I've changed `n` as well